test/language/statements/class/dstr-private-gen-meth-static-dflt-obj-ptrn-rest-val-obj.js

```diff
@@ -4,8 +4,7 @@
 /*---
 description: Rest object contains just unextracted data (private static class expression generator method (default parameter))
 esid: sec-runtime-semantics-bindingclassdeclarationevaluation
-features: [object-rest, class, class-static-methods-private, generators, destructuring-binding,
-  default-parameters]
+features: [object-rest, class, class-static-methods-private, generators, destructuring-binding,  default-parameters]
```

Comment:
Nit: this leaves double spaces (before `default-parameters`). Maybe adjust the regex to trim `\n` followed by `\s*` into a single space instead?